GlyssenShared/GlyssenShared.csproj

```diff
@@ -5,6 +5,10 @@
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
```

Comment:
I'd put the relevant information here in the .csproj file and let msbuild generate assembly info (and remove `AssemblyInfo.cs`) because that puts all information in one place without having to duplicate e.g. `Authors`.